scripts/retrieve_databundle_light.py

```diff
@@ -617,6 +701,36 @@ def datafiles_retrivedatabundle(config):
 
     return listoutputs
 
+    def merge_hydrobasins_shape(config):
+        basins_path = config_bundles["bundle_hydrobasins"]["destination"]
+
+        files_in_dir = [
+            f
+            for f in os.listdir(basins_path)
+            if os.path.isfile(os.path.join(basins_path, f))
+        ]
+
+        hydrobasins_level = snakemake.config["renewable"]["hydro"]["resource"][
+            "hydrobasins_level"
+        ]
+        if hydrobasins_level <= 9:
+            hydrobasins_level = "0" + str(hydrobasins_level)
+        else:
+            hydrobasins_level = str(hydrobasins_level)
+
+        regex_to_look_for = re.compile(".*_lev" + hydrobasins_level + "_v1c.shp$")
+        files_to_merge = list(filter(regex_to_look_for.match, files_in_dir))
```

Comment:
Probably, we could exploit the Path object here and do something like:
```
mask_file = "hybas_*_lev{:02d}_v1c.shp".format(int(hydrobasins_level))
files_to_merge = Path(basins_path).glob(mask_file)
```

Reply:
Perfect :)
Implemented, many thanks for the suggestions!